### ISSUE TITLE
No screen Reader Speech on brackets in MuseScore 4.6, poor speech in 4.5

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31756]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In MuseScore 4.6 the screen reader is not announcing the presence of brackets around elements and is not navigating to them.

STEPS:
1.  Navigate to a note and press Shift 9 to add a parenthesis.  Notice the screen reader announces "Add parenthesis to element", but the user does not know if it has actually been added as in 4.6 the screen reader is often announcing the name of a key press when it can't perform an action.
2. Arrow off and on this note and notice the speech does not announce the presence of a bracket.  Strangely the note is played slightly softer.
3. Now use Alt arrows to move across this area and notice the bracket element is skipped.
4. Move to another note that has an accidental and choose add bracket to accidental from the accidentals palette.  
5. Repeat Steps 2 & 3 arrowing and Alt arrowing across the note with the reminder accidental in brackets.  Notice there is no announcement of any sort and the bracket is skipped.  It plays as usual and the screen reader user has no way to confirm its presence or know it is there if reading a score.

In MuseScore 4.5 these did read, but the result was verbose and confusing as below:
1.  Add a note Bb to the score and press Shift 9 to add parenthesis.
2. Arrow across it and you hear "Note B flat 4 crotchet, bar 2 beat 4  accidental flat symbol open parenthesis symbol closing parentheses."  There are two issues with this, it does not tell the user where the parenthesis is, is it around the whole note + accidental, or just the accidental?  Second it is too long, there is no need to hear opening and closing parenthesis or symbol, just parenthesis.
3. Now use Alt arrow to move across this and you will hear two silent presses after the note, one for opening and one for closing parenthesis.  It would be better if it only moved to one of these and said "parenthesis".
Note also there needs to be a clear short way of the user knowing which element the bracket is around. Maybe something like "parenthesis around note", "parenthesis around accidental", etc.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.5

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
